solutions/src/main/java/ru/sberbank/school/task08/SerializableManager.java

```diff
@@ -16,18 +17,38 @@ public SerializableManager(@NonNull String filesDirectoryPath) {
     }
 
     @Override
-    public void initialize() {
-        throw new UnsupportedOperationException("Implement me!");
-    }
+    public void initialize() {}
 
     @Override
     public void saveGame(String filename, Savable gameState) throws SaveGameException {
-        throw new UnsupportedOperationException("Implement me!");
+        try (FileOutputStream fos = new FileOutputStream(filesDirectory + filename);
+            ObjectOutputStream out = new ObjectOutputStream(fos)){
+            out.writeObject(gameState); }
```

Comment:
&#39;}&#39; at column 41 should have line break before.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a543546970>*